### ISSUE TITLE
`npm run svelto:dev` fails with Error: Unhandled "error" event. ([object Object])

### ISSUE THREAD

**Issue by jankeromnes** [fabiospampinato/notable#253]
Hello! Thank you for making a cool-looking notes app. 😄 👍 Today I wanted to try building `notable` in https://gitpod.io (disclaimer: I work for Gitpod) by following the steps in [contributing](https://github.com/fabiospampinato/notable#contributing), but it didn't work, and I'm not quite sure how to investigate this problem or fix it.

---

- **OS Version**: Ubuntu 18.10
- **Notable Version**: master @ 0eee7e273ec9f7288590c623f44884670ede44e8

### Current behavior

Steps to reproduce:

1. I checked out `notable` in Gitpod using this URL: https://gitpod.io/#https://github.com/fabiospampinato/notable
2. I ran `npm install`, which worked
3. I ran `npm run svelto:dev`, which worked up to a point, but then failed with:

> [...]
> [13:49:02] Finished 'build-css' after 5ms
> [13:49:02] Finished 'build-style' after 2s
> [13:49:02] Finished 'build' after 2s
> Bundled into "/workspace/notable/src/renderer/template/dist" (1.4 MB) in 2s
> /workspace/notable/node_modules/async-done/index.js:18
>     throw err;
>     ^
> 
> Error: Unhandled "error" event. ([object Object])
>     at Gulp.emit (events.js:186:19)
>     at Object.error (/workspace/notable/node_modules/undertaker/lib/helpers/createExtensions.js:61:10)
>     at handler (/workspace/notable/node_modules/now-and-later/lib/mapSeries.js:43:14)
>     at f (/workspace/notable/node_modules/once/once.js:25:25)
>     at f (/workspace/notable/node_modules/once/once.js:25:25)
>     at tryCatch (/workspace/notable/node_modules/async-done/index.js:24:15)
>     at done (/workspace/notable/node_modules/async-done/index.js:40:12)
>     at Domain.onError (/workspace/notable/node_modules/async-done/index.js:51:5)
>     at Object.onceWrapper (events.js:315:30)
>     at emitOne (events.js:116:13)
>     at Domain.emit (events.js:211:7)
>     at Domain._errorHandler (domain.js:118:23)
>     at process._fatalException (bootstrap_node.js:388:33)

### Expected behavior

I expected `npm run svelto:dev` to finish without errors.

Please let me know if I'm doing something wrong here.

Also, I'm happy to help troubleshoot and fix this issue: please let me know what I can do (e.g. is there a "debug" build option in order to get more details on what that mysterious `[object Object]` error event is all about)?

**Comment by jankeromnes** [comment]
Update: It looks like the error is being thrown from the `undertaker` NPM module:

https://github.com/gulpjs/undertaker/blob/c2373f6df3e4e5d7354b938cc0303df6c91949e4/lib/helpers/createExtensions.js#L61-L68

I've added a `console.error('ERROR', error)` call to their error handler, and it gave me this:

```
ERROR { Error: not found: notify-send
    at getNotFoundError (/workspace/notable/node_modules/which/which.js:13:12)
    at Function.whichSync [as sync] (/workspace/notable/node_modules/which/which.js:134:9)
    at NotifySend.notify (/workspace/notable/node_modules/node-notifier/notifiers/notifysend.js:65:27)
    at P (/workspace/notable/node_modules/pify/index.js:32:6)
    at new Promise (<anonymous>)
    at /workspace/notable/node_modules/pify/index.js:6:9
    at ret (/workspace/notable/node_modules/pify/index.js:56:76)
    at Object.send (/workspace/notable/node_modules/pacco/src/utilities/notification.js:34:60)
    at task (/workspace/notable/node_modules/pacco/src/tasks/build/notification.js:21:25)
    at bound (domain.js:301:14)
    at runBound (domain.js:314:12)
    at asyncRunner (/workspace/notable/node_modules/async-done/index.js:55:18)
    at _combinedTickCallback (internal/process/next_tick.js:132:7)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9) code: 'ENOENT' }
```

If that's the root cause of my error, then it has nothing to do with `notable` directly and should probably be fixed upstream in one of the dependencies. It's the `node-notifier` module which seems to throw when there is no `notify-send` binary found, but maybe one of the callers should catch that exception and fail more gracefully.

**Comment by fabiospampinato** [comment]
@jankeromnes thanks for looking into this! This is problably a problem with [pacco](https://github.com/fabiospampinato/pacco), the bundler used to bundle Svelto, maybe there's a missing dependency? We are showing a notification once the compilation is completed (I think there's a `--no-notification` flag available as a temporary workaround), but I don't think it should throw an error if it's impossible to do that.

**Comment by fabiospampinato** [comment]
This issue is now fixed in pacco@2.1.3.
